lib/streamlit/elements/file_uploader.py

```diff
@@ -187,6 +192,22 @@ def serialize_file_uploader(
             serializer=serialize_file_uploader,
         )
 
+        ctx = get_report_ctx()
+        if ctx is not None and widget_value:
```

Comment:
Nit: don't test against truthy/falsy for widget_value: `if ctx is not None and widget_value is not None`

(I wish we had a linter for this - we enforce this on the frontend but not on the backend)

Reply:
:+1: fixed

Reply:
Ah, shoot. I seem to have made the change locally and forgotten to push it up before clicking merge. Will make a followup PR to actually fix this right now.